Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.9-slim
+
+# Keeps Python from generating .pyc files in the container
+ENV PYTHONDONTWRITEBYTECODE 1
+# Turns off buffering for easier container logging
+ENV PYTHONUNBUFFERED 1
+
+# Install dev dependencies
+RUN apt-get update
+RUN apt-get install -y libpq-dev gcc
+
+# Install poetry
+RUN pip install -U pip
+RUN pip install "poetry==1.1.14"
+
+COPY ./performance_manager /performance_manager/
+WORKDIR /performance_manager/
+RUN poetry config virtualenvs.create false
+RUN poetry install --no-dev --no-interaction --no-ansi
```

Comment:
without a `CMD` or `ENTRYPOINT` the container exits immediately after starting. Is this the expected behavior?

Reply:
for now yeah. after you stand the containers up, you run

`docker-compose run --rm performance_manager python thing_to_run.py` and it will kick off the job. once i get a better sense of how ECS Tasks work, I'll get this portion lined up correctly.